main.go

```diff
@@ -22,9 +27,28 @@ var (
 )
 
 func main() {
+	var dbHost string;
```

Comment:
Others disagree: http://12factor.net/config But I really don't think this PR is the place to discuss the issue.